src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -196,6 +181,57 @@ public async Task InvokeAsync(HttpContext context)
             }
         }
 
+        protected virtual async ValueTask<bool> HandleDeserializationErrorAsync(HttpContext context, Exception ex)
+        {
+            await WriteErrorResponseAsync(
```

Comment:
`WriteErrorResponseAsync` is private so no one can utilize it as _default path_.

Reply:
Made protected and simplified arguments.